src/index.js

```diff
@@ -2,0 +1,43 @@
+// 'use strict';
+
+// import express from 'express';
+// import cors from 'cors';
+
+// const app = express();
+
+// app.use(cors({ origin: '*' }));
```

Comment:
The middleware for enabling CORS is commented out. Uncomment this line to allow cross-origin requests.